src/locales/en.json

```diff
@@ -247,23 +247,14 @@
     },
     "tips": "Tips"
   },
-  "privacy": {
-    "title": "You'll have to upload a photo of your identity document",
-    "item_1": "All your document details must be visible",
-    "item_2": "Your document must be in colour",
-    "item_3": "Avoid light reflections",
-    "small_print": "By continuing, you agree to the <terms>Onfido Terms of Use</terms> and understand that your information, including your facial identifiers, will be processed in accordance with the <privacy>Onfido Privacy Policy</privacy>",
-    "decline": "Decline",
-    "continue": "Continue"
-  },
   "webcam_permissions": {
     "allow_access": "Allow camera access",
     "enable_webcam_for_selfie": "When prompted, you must enable camera access to continue",
     "click_allow": "We cannot verify you without using your camera",
     "allow": "Allow",
     "why": "Why do I need to do this?",
     "if_denied": "If you deny camera access, you won't be able to take pictures and complete verification process.",
-    "enable_webcam": "Enable camera",
+    "enable_webcam": "Enable webcam",
```

Comment:
I think this string is shared between desktop and mobile. IMO, "Enable webcam" doesn't sound good on mobile.

Reply:
sorry didn't change it on the spreadsheet, should I ? @raemarielee

Reply:
On the spreadsheet `webcam_permissions.allow` has been renamed to "Enable camera" (originally "Allow"), don't see it in use in the code as the button's label is currently mapped to `webcam_permissions.enable_webcam`. Probably should be using that other string map instead.

Reply:
fixed